packages/toolkit/src/auth/ui/vue/show.ts

```diff
@@ -744,6 +745,12 @@ export const showManageConnections = Commands.declare(
         if (!isServiceItemId(serviceToShow)) {
             serviceToShow = undefined
         }
+
+        if (DevSettings.instance.isNewLoginEnabled()) {
+            void vscode.commands.executeCommand('setContext', 'aws.explorer.showAuthView', true)
```

Comment:
why not `await`? Then purpose of that lint rule is not to ignore it, but to avoid race conditions.

Reply:
Fixed in newer commits